module1/presentation_array.md

```diff
@@ -63,7 +63,7 @@ int main() {
 
 ___
 
-## Zadanie
+## Zadanie 3
```

Comment:
we don't want to have numbers there, because the content of this files may be served in a different order. It's just "Zadanie" :)
```suggestion
## Zadanie
```